api/base/exceptions.py

```diff
@@ -23,12 +24,14 @@ def json_api_exception_handler(exc, context):
                 else:
                     if isinstance(value, list):
                         for reason in value:
-                            errors.append({'detail': reason, 'meta': {'field': key}})
+                            errors.append({'source': {'pointer': '/data/attributes/' + key}, 'detail': reason})
```

Comment:
This can be rewritten as a list comprehension to be more concise:
```
errors.extend([{'source': {'pointer': '/data/attributes/' + key}, 'detail': reason}
                        for reason in value])
```

Consider renaming the variables `key` and `value` to something a bit more descriptive of the dictionary contents as well.